### DISCUSSION TITLE
Custom links to directly import into Draw Things app?

### DISCUSSION THREAD

**Discussion by liuliu** [civitai/civitai#85] [Ideas]
Hi! My app recently supported import custom models. Many users use your service to discover new models and import into the app: https://twitter.com/drawthingsapp

It would be great if we can figure out some kind of custom links: https://developer.apple.com/ios/universal-links/ such that browsing the site, and tap a link would bring people to the app, ready to import the model. It could even bring some metadata to prefill, making it easier.

Let me know how do you think. I have domain https://drawthings.ai/, so if need to support universal link, that's a starting point.

**Comment by JustMaier** [comment]
I'd love this! Don't know if you're a discord user, but if you are, jump into our server and give me a shout out so that we can talk through how this could work!

[**Discord Server**](https://discord.com/invite/UwX5wKwm6c)

If Discord doesn't work for you, let me know what does so that we can jam on the idea :)

**Comment by JustMaier** [comment]
**This is live!**
![image](https://user-images.githubusercontent.com/607609/210708084-fbfee3f5-25b9-4761-a25f-55758db50a05.png)

Thanks @liuliu for all your hard work adding support for this! It's awesome that anyone with a modern iOS device or MacOS can now enjoy all of the models so easily.

**Comment by IIIIIIIllllllllIIIII** [comment]
do i have to be logged in to use this feature? i only get that no services are supported for mst models... and for draw things it should be available for all models since its local and i would just download it... ?!

**Comment by IIIIIIIllllllllIIIII** [comment]
![image](https://github.com/user-attachments/assets/99eda031-3cb6-4b7d-8fa2-029fd10f803d)
